fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java

```diff
@@ -120,6 +128,20 @@ private WorkloadGroup(long id, String name, Map<String, String> properties, long
             this.cpuHardLimit = Integer.parseInt(cpuHardLimitStr);
             this.properties.put(CPU_HARD_LIMIT, cpuHardLimitStr);
         }
+        if (properties.containsKey(SPILL_THRESHOLD_LOW_WATERMARK)) {
+            String lowWatermarkStr = properties.get(SPILL_THRESHOLD_LOW_WATERMARK);
+            if (lowWatermarkStr.endsWith("%")) {
+                lowWatermarkStr = lowWatermarkStr.substring(0, lowWatermarkStr.length() - 1);
```

Comment:
Has to check the value :
1. could be parsed to int
2. the value is between -1,100

Reply:
Value check in WorkloadGroup.checkProperties